_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -0,0 +1,483 @@
+---
+id: ffbc54
+name: No keyboard shortcuts use only printable characters
+rule_type: atomic
+description: |
+  This rule checks that if keyboard shortcuts are implemented using only printable characters, then there is a mechanism to disable the shortcut, or to remap the shortcut to use one or more non-printable characters keys, or the shortcut for a user interface component is only available when that component has focus.
+accessibility_requirements:
+  wcag21:2.1.4: # Character Key Shortcuts (A)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+acknowledgments:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [keyboard event][] for which all is true: 
+- the event's attribute `key` is a [printable character][] key; and
+- the event's method `getModifierState` returns `false` for any of the [valid modifier keys][]; and
+- the event is [dispatched][] to an [event target][] within a [HTML document][].
```

Comment:
Not sure that the "dispatched" applicability is still needed. I think it was introduced before we had all the "change in content" stuff, as a way to say "the event does something". We might get rid of it altogether and consider that the event which is not dispatched will cause no change in content and thus pass the rule one way or another.